app/assets/javascripts/sitewide/file_preview.js

```diff
@@ -349,8 +353,12 @@
             }
           }
         } else {
+          modal.find('.file-edit-link').css('display', 'none');
           modal.find('.file-preview-container').html(data['preview-icon']);
         }
+        if (readOnly) {
+          modal.find('#wopi_file_edit_button').remove();
```

Comment:
I don't know what the requirements are but usually the edit button is disabled and the hover on shows the reason for not being able to edit.

Reply:
This parameter is used to disable file editing on the whole page(like protocol preview).